src/server/game/Autobroadcast/AutobroadcastMgr.cpp

```diff
@@ -71,6 +71,39 @@ void AutobroadcastMgr::LoadAutobroadcasts()
     LOG_INFO("autobroadcast", " ");
 }
 
+void AutobroadcastMgr::LoadAutobroadcastsLocalized()
+{
+    uint32 oldMSTime = getMSTime();
+    uint32 realmId = sConfigMgr->GetOption<int32>("RealmID", 0);
+
+    if (_autobroadcasts.size() == 0)
+        return;
+
+    LoginDatabasePreparedStatement* stmt = LoginDatabase.GetPreparedStatement(LOGIN_SEL_AUTOBROADCAST_LOCALIZED);
+    stmt->SetData(0, realmId);
+    PreparedQueryResult result = LoginDatabase.Query(stmt);
+
+    if (!result)
+    {
+        LOG_WARN("autobroadcast", ">> Loaded 0 localized autobroadcasts definitions. DB table `autobroadcast_localized` is empty for this realm!");
+        return;
+    }
+
+    uint32 count = 0;
+    do
+    {
+        Field* fields = result->Fetch();
+        uint8 id = fields[0].Get<uint8>();
+        LocaleConstant locale = GetLocaleByName(fields[1].Get<std::string>());
+        std::string message = fields[2].Get<std::string>();
+
+        _localizedAutobroadcasts[id][locale] = message;
+        ++count;
+    } while (result->NextRow());
+
+    LOG_INFO("autobroadcast", ">> Loaded {} Localized Autobroadcast Definitions in {} ms", count, GetMSTimeDiffToNow(oldMSTime));
```

Comment:
You could remove count and replace it with _localizedAutobroadcasts.size() I think